agent/consul/server_test.go

```diff
@@ -209,19 +209,24 @@ func newServer(c *Config) (*Server, error) {
 	if w == nil {
 		w = os.Stderr
 	}
+	logger2 := hclog.New(&hclog.LoggerOptions{
+		Name:   c.NodeName,
+		Level:  hclog.Debug,
+		Output: w,
+	})
 	consulLogger := hclog.New(&hclog.LoggerOptions{
```

Comment:
Why have a `logger2` and a `consulLogger` both defined with the same content?

Reply:
No reason at all!